src/PlaywrightSharp/Helpers/JsonExtensions.cs

```diff
@@ -73,6 +82,7 @@ internal static JsonSerializerOptions GetNewDefaultSerializerOptions(bool ignore
                 {
                     new FlexibleStringEnumConverter<ResourceType>(ResourceType.Unknown),
                     new JsonStringEnumMemberConverter(JsonNamingPolicy.CamelCase),
+                    new EvaluateArgumentConverter(),
```

Comment:
We can't add this to all serializer calls.

Reply:
why? are there any side effect?

Reply:
Yes, tests are failing because of that.
The EvaluateArgumentCoverter should be used only when we need to serialize evaluate arguments. That folk does crazy stuff.